<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automata Celular v2.0</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,30">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_Title">
           <property name="font">
            <font>
             <family>Futura</family>
             <pointsize>22</pointsize>
             <italic>false</italic>
             <underline>false</underline>
             <strikeout>true</strikeout>
            </font>
           </property>
           <property name="text">
            <string>M A R D - SYSTEM</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="tabPosition">
            <enum>QTabWidget::West</enum>
           </property>
           <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <property name="elideMode">
            <enum>Qt::ElideRight</enum>
           </property>
           <property name="movable">
            <bool>true</bool>
           </property>
           <property name="tabBarAutoHide">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Cell</string>
            </attribute>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>91</width>
               <height>16</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Celulas vivas:</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_AliveCells">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>20</y>
               <width>58</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>60</y>
               <width>91</width>
               <height>16</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Generaci√≥n:</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_Generation">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>60</y>
               <width>58</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Colores</string>
            </attribute>
            <widget class="QPushButton" name="pushButton_Acolor">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>61</width>
               <height>32</height>
              </rect>
             </property>
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>81</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Celulas vivas</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>101</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Celulas muertas</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_Bcolor">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>90</y>
               <width>61</width>
               <height>32</height>
              </rect>
             </property>
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_Acolor">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>10</y>
               <width>21</width>
               <height>16</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>true</bold>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_Dcolor">
             <property name="geometry">
              <rect>
               <x>110</x>
               <y>70</y>
               <width>21</width>
               <height>16</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Rule</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Smin:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox_Smin">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>8</number>
                   </property>
                   <property name="value">
                    <number>2</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Smax:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox_Smax">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>8</number>
                   </property>
                   <property name="value">
                    <number>3</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Nmin:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox_Nmin">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>8</number>
                   </property>
                   <property name="value">
                    <number>3</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Nmax:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox_Nmax">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>8</number>
                   </property>
                   <property name="value">
                    <number>3</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_Rule">
               <property name="text">
                <string>Set Rule</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_6">
            <attribute name="title">
             <string>File</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QScrollArea" name="scrollArea">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>173</width>
                  <height>167</height>
                 </rect>
                </property>
                <widget class="QWidget" name="layoutWidget">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>10</y>
                   <width>124</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>Cargar archivo</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="tool_file">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="layoutWidget">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>60</y>
                   <width>131</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>Guardar archivo</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="tool_fileSave">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Tama√±o del Universo</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QSpinBox" name="spinBox_Size">
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>5000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_Size">
               <property name="text">
                <string>Set Size</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_Start">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Stop">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Pause">
             <property name="text">
              <string>Pause</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Rotar</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSlider" name="Rotate">
             <property name="maximum">
              <number>360</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Rotate">
             <property name="text">
              <string>0¬∞</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Zoom</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSlider" name="Zoom">
             <property name="minimum">
              <number>-1000</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Zoom">
             <property name="text">
              <string>0%</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>AC</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Density</string>
      </attribute>
      <widget class="QDockWidget" name="dockWidget_Density">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>12</y>
         <width>251</width>
         <height>181</height>
        </rect>
       </property>
       <property name="windowTitle">
        <string>Density</string>
       </property>
       <widget class="QWidget" name="dockWidgetContents"/>
      </widget>
      <widget class="QPushButton" name="pushButton_Chart">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>101</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Display Chart</string>
       </property>
      </widget>
      <widget class="QSlider" name="Zoom_Chart">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>440</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QDockWidget" name="dockWidget_Log10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>251</width>
         <height>151</height>
        </rect>
       </property>
       <property name="windowTitle">
        <string>Log10 Density</string>
       </property>
       <widget class="QWidget" name="dockWidgetContents_2"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Attractors</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_Nodes">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>491</width>
         <height>271</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>324</y>
         <width>221</width>
         <height>93</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Tama√±o del Espacio</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QSpinBox" name="spinBox_SizeGrid">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>6</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Calculate">
            <property name="text">
             <string>Calculate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>390</y>
         <width>219</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Guardar archivo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_fileSave2">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
